### ISSUE TITLE
Better horizontal scrolling

### ISSUE THREAD

**Issue by brielov** [amit9838/mousam#43]
Currently, the only way to scroll through the horizontal list is by using the scrollbar (which is tiny and sometimes difficult to grab). This is fine but it would be nice to be able to scroll using the scroll wheel or/and click-and-drag.

![image](https://github.com/amit9838/weather/assets/42145089/d52ac501-5651-47cf-8372-a7c496eedad8)

**Comment by amit9838** [comment]
Can mention you device info?

**Comment by brielov** [comment]
@amit9838  Yes, sorry. I'm on a Linux desktop with a 32 inch, 165hz monitor.

# System Details Report
---

## Report details
- **Date generated:**                              2024-01-15 14:00:56

## Hardware Information:
- **Hardware Model:**                              Gigabyte Technology Co., Ltd. X570S AORUS ELITE
- **Memory:**                                      64.0 GiB
- **Processor:**                                   AMD Ryzen™ 9 5900X × 24
- **Graphics:**                                    AMD Radeon™ RX 6900 XT
- **Disk Capacity:**                               1.5 TB

## Software Information:
- **Firmware Version:**                            F6
- **OS Name:**                                     Fedora Linux 39 (Workstation Edition)
- **OS Build:**                                    (null)
- **OS Type:**                                     64-bit
- **GNOME Version:**                               45.3
- **Windowing System:**                            Wayland
- **Kernel Version:**                              Linux 6.4.9-200.fc38.x86_64

**Comment by amit9838** [comment]
thanks

**Comment by Danversion** [comment]
Click + drag could be useful for some people who prefer it but as @amit9838 said holding down shift changes scroll to horizontal which I think is better than click + drag.
